moviepy/editor.py

```diff
@@ -1,133 +1,63 @@
 """
-This file is meant to make it easy to load the main features of
-MoviePy by simply typing:
+This file is meant to make it easy to load the features of
+MoviePy that you will use for live editing by simply typing:
 
 >>> from moviepy.editor import *
 
-In particular it will load many effects from the video.fx and audio.fx
-folders and turn them into VideoClip methods, so that instead of
->>> clip.fx( vfx.resize, 2 ) or equivalently vfx.resize(clip, 2)
-we can write
->>> clip.resize(2)
-
-It also starts a PyGame session (if PyGame is installed) and enables
-clip.preview().
+- Starts a pygame session to enable ``clip.show()`` and ``clip.preview()`` if pygame is installed
+- Enables ``clip.ipython_display()`` if in an IPython Notebook
+- Allows the use of ``sliders`` if Matplotlib is installed
 """
 
-__all__ = [
```

Comment:
Could you maintain `__all__` here, removing those imports that now are not in the file?